src/uwtools/utils/file.py

```diff
@@ -30,19 +30,20 @@ class FORMAT:
     _ini: str = "ini"
     _jinja2: str = "jinja2"
     _nml: str = "nml"
+    _sh: str = "sh"
```

Comment:
`uwtools.cli` contains the line
```
FORMATS = [FORMAT.ini, FORMAT.nml, FORMAT.yaml]
```
Does this need to be updated to include `FORMAT.sh`?

Reply:
AH, I knew I would miss one, thanks!